pkgs/development/libraries/pinocchio/default.nix

```diff
@@ -38,20 +39,31 @@ stdenv.mkDerivation rec {
   ] ++ lib.optionals (!pythonSupport) [
     boost
     eigen
+    hpp-fcl
   ] ++ lib.optionals pythonSupport [
     python3Packages.boost
     python3Packages.eigenpy
+    python3Packages.hpp-fcl
   ];
 
-  cmakeFlags = lib.optionals (!pythonSupport) [
+  cmakeFlags = [
+    "-DBUILD_WITH_COLLISION_SUPPORT=ON"
+  ] ++ lib.optionals (pythonSupport) [
+    "-DBUILD_WITH_LIBPYTHON=ON"
+  ] ++ lib.optionals (!pythonSupport) [
     "-DBUILD_PYTHON_INTERFACE=OFF"
   ];
 
+  doCheck = true;
```

Comment:
```suggestion
  doCheck = !stdenv.isDarwin;
```

Reply:
Oh, wait, this is disabling tests on pinochio. Is this really what you intended ? I assumed it was for example-robot-data.

Reply:
Yes, since five tests failed on ofborg. Haven't looked into it, though.